### DISCUSSION TITLE
Change `src` to `contracts`

### DISCUSSION THREAD

**Discussion by andreivladbrg** [sablier-labs/evm-monorepo#655] [Ideas]
In the latest npm packages release for `v2-core` and `v2-periphery` we addressed an [issue](https://github.com/sablier-labs/v2-core/issues/646) faced by our integrators when importing `@prb/math` in `v2-core` and `@prb/proxy` in `v2-periphery`.

We fixed it by removing `src` in the remappings file:

https://github.com/sablier-labs/v2-core/blob/456744002f62d0627f10b958d41937ed1e68d95e/remappings.txt#L2

https://github.com/sablier-labs/v2-periphery/blob/7283580fadc2a7a1677b3dd53f3d76b0ff967fdd/remappings.txt#L2

The question I want to ask is: would it have been better if we had changed the `src` dir to `contracts`?

Ofc, the `foundry.toml` would need to be updated accordingly: `src = "contracts"`

**Comment by PaulRBerg** [comment]
Interesting idea but I am tentatively against it.

The production contracts directory is named "contracts" in OpenZeppelin and Uniswap and other projects due to historical reasons, namely, this was the name of the default directory structure generated by Truffle and Hardhat.

Foundry switched to `src` because Foundry is all about testing with Solidity. The files in `test` are contracts, too. Therefore, I would be inclined to stick with `src` even if it means using the import path in the production contracts.

I also think this is a case of "de minimis" - we have other, more important issues to think about.

**Reply by andreivladbrg** [reply]
> we have other, more important issues to think about.

Agree, we can leave it as it is for the time being.
